pkg/asset/installconfig/platform.go

```diff
@@ -220,11 +220,38 @@ func (a *Platform) libvirtPlatform() (*asset.State, error) {
 		},
 		EnvVarName: "OPENSHIFT_INSTALL_LIBVIRT_URI",
 	}
-	uri, err := prompt.Generate(nil)
+	uri, err := uriPrompt.Generate(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	imagePrompt := asset.UserProvided{
+		Question: &survey.Question{
+			Prompt: &survey.Input{
+				Message: "Image",
+				Help:    "URI for the os image.",
+				Default: platform.DefaultMachinePlatform.Image,
+			},
+			Validate: survey.ComposeValidators(survey.Required, func(ans interface{}) error {
```

Comment:
Spin this off into a local helper function to share with the libvirtd URI query?